struct_test.go

```diff
@@ -682,10 +682,10 @@ func TestStructOmitEmptyForTag(t *testing.T) {
 }
 
 type structOmitEmptyForMultipleTags struct {
-	A int      `db:"aa" fieldopt:"omitempty, omitempty(patch,patch2),withquote" fieldtag:"patch,patch2"`
+	A int      `db:"aa" fieldopt:"omitempty, omitempty(patch, patch2),withquote" fieldtag:"patch, patch2 "`
```

Comment:
here it tests an error that was previous to this PR: fieldtags couldn't have spaces because they were considered part of the tag.( it was treated with a split, with no trim)

Reply:
It's fine to leave spaces in tag.